### ISSUE TITLE
DFU error message not visible

### ISSUE THREAD

**Issue by philips77** [NordicSemiconductor/Android-Nordic-Thingy#5]
An update of my Thingy failed at some point. The error received was `Executing object failed (error 5): INVALID OBJECT` (that's actually not important). 
The app did show the error, but the message was light gray with light gray X put on the white background. The iOS Thingy app (amazing job!), on the other hand, shows a red X and black error message which is much better visible and you immediately feel the sadness of the moment. You want to cry or try again. On Android it's more like: "Whaaat? It stopped and doesn't go further... why? Oh, there is an error!".

In conclusion: the error reporting in DFU should be improved.

**Comment by roshanrajaratnam** [comment]
Thanks for reporting this issue. This has been fixed in the latest release v1.3.4
